### PR TITLE
Update committer guide

### PR DESCRIPTION
**Pull request by holdenk**
Update the committer guide with the new policy we voted on for when to commit.

### CODE REVIEW

**Comment by holdenk** [comment]
Change is:
image

File: site/committers.html [hunk 1]
```diff
@@ -613,9 +629,9 @@ <h4>Merge Script</h4>
 
 <p>If you want to amend a commit before merging – which should be used for trivial touch-ups – 
 then simply let the script wait at the point where it asks you if you want to push to Apache. 
-Then, in a separate window, modify the code and push a commit. Run <code class="highlighter-rouge">git rebase -i HEAD~2</code> and 
+Then, in a separate window, modify the code and push a commit. Run <code class="language-plaintext highlighter-rouge">git rebase -i HEAD~2</code> and
```
Comment (srowen) [inline site/committers.html:632]: I think these other changes are coming from differences in jekyll version - are you using 4.1.x? Not a big deal but I am not sure everyone is and I usually revert the noise changes and one day we'll standardize!

**Review by srowen — COMMENTED** [review]
Looks good content-wise

File: site/committers.html (hunk 1 above)
Comment (holdenk) [reply site/committers.html:632]: I'm running Jekyll 4.1.1

Comment (srowen) [reply site/committers.html:632]: Ha OK then your results are 'correct'; it had been previously generated with jekyll 3.x. I think it's fine to choose to revert those, or commit them. Really we should regenerate everything with jekyll 4.1.x and really ask people stick to the latest, and be done with it.

File: committers.md [hunk 2]
```diff
@@ -132,6 +132,27 @@ In particular, if you are working on an area of the codebase you are unfamiliar
 Git history for that code to see who reviewed patches before. You can do this using 
 `git log --format=full <filename>`, by examining the "Commit" field to see who committed each patch.
 
+<h3>When to commit/merge a pull request</h3>
+
+PRs shall not be merged during active, on-topic discussion unless they address issues such as critical security fixes of a public vulnerability. Under extenuating circumstances, PRs may be merged during active, off-topic discussion and the discussion directed to a more appropriate venue. Time should be given prior to merging for those involved with the conversation to explain if they believe they are on-topic.
```
Comment (dongjoon-hyun) [inline committers.md:137]: In this context, `Time should be given prior to merging`, can we have an example like 72 hour explicitly?

Comment (holdenk) [reply committers.md:137]: Ah, I don't think we can change the wording anymore since we voted on it.

Comment (holdenk) [reply committers.md:137]: Or well we can but I'd prefer to take it to another vote first before changing the policy.

Comment (srowen) [reply committers.md:137]: I think minor clarifications are entirely fine. No need for a new vote.

Comment (holdenk) [reply committers.md:137]: Sounds good to me then. So we can do something like "often at least 72 hours"?

Comment (dongjoon-hyun) [reply committers.md:137]: +1 for `often at least 72 hours`

Comment (HyukjinKwon) [reply committers.md:137]: I think we can just say "for example 72 hours". That's [the example in the lazy consensus](https://www.apache.org/foundation/voting.html#LazyConsensus), and I believe we're rather clarifying and reiterating what ASF norms wrote there.

**Comment by HyukjinKwon** [comment]
Out of curiosity, why is it only in Spark website? I thought the general issue is https://www.apache.org/foundation/voting.html page itself isn't clear and we targeted to fix that page.

**Comment by srowen** [comment]
Sure bonus points for that. I think PMCs have this latitude to customize and specialize that generic guidance to their specific situations.

**Comment by HyukjinKwon** [comment]
The doc here looks good as we voted and discussed.

**Review by srowen — APPROVED** [review]
Feel free to commit I'd say!

**Comment by holdenk** [comment]
Ok I'm going to go ahead and commit, and if @dongjoon-hyun & @HyukjinKwon want to update with the 72 hours and can agree on the wording we can do that in a follow up PR.

**Review by dongjoon-hyun — APPROVED** [review]
Sure. +1, LGTM, @holdenk . Sorry for making a delay here.

**Comment by dongjoon-hyun** [comment]
Please merge this AS-IS.

**Pull request closed without merging**